ner_v1/constant.py

```diff
@@ -51,6 +51,18 @@
 # ********************** constant used to define dict type in data dictionary *********************
 DICTIONARY_DATA_VARIANTS = 'variants'
 
+# **********************constants used for number detection************************************
+
+DIGIT_UNITS = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"]
+NUMERIC_VARIANTS = DIGIT_UNITS + [
+    "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen",
+    "sixteen", "seventeen", "eighteen", "nineteen", "and",
+    "twenty", "thirty", "forty", "fifty", "sixty", "seventy", "eighty", "ninety",
+    "hundred", "thousand", "million", "billion", "trillion", "fourty", "ninty"
+]
+PARAMETER_MIN = 'min'
+PARAMETER_MAX = 'max'
```

Comment:
Can we change these to `PARAMETER_MIN_DIGITS = min_number_digits` and `PARAMETER_MAX_DIGITS = max_number_digits`

Just avoids confusion by being explicit

Reply:
sure